assets/css/admin/upgrades/_v3.scss

```diff
@@ -0,0 +1,15 @@
+#edd-migration-progress {
+	.dashicons-yes {
+		color: green;
+	}
+	.dashicons-update:before {
+		animation: rotation 2s infinite linear;
+		display: block;
+	}
+}
+
+#edd-v3-migration-remove-legacy-data-submit-wrap {
```

Comment:
On smaller screens, the `.wp-core-ui .button` has a `margin-bottom: 4px;` which is throwing off the vertical alignment slightly. If we could set that to have a 0 margin it would work on all screens.